src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -18,7 +18,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ImportGroup>
 
   <PropertyGroup>
-    <DefaultExcludes>bin\**;obj\**</DefaultExcludes>
+    <DefaultExcludes>bin\**;obj\**;packages\**</DefaultExcludes>
```

Comment:
@livarcocc The only case where adding `**\*.xproj` here would have an effect is if there is an `.xproj` in under the `compiler\resources` folder.  Is there any normal scenario where that would happen?